clang/test/Driver/linker-wrapper-passes.ll

```diff
@@ -0,0 +1,77 @@
+; Check various clang-linker-wrapper pass options after -offload-opt.
```

Comment:
Hm, is this really the only LLVM-IR file in the Driver directory? I guess it makes sense, though you could probably just do what the other linker wrapper tests do and use `clang-cc1` to directly get some random IR to use.

Reply:
Yeah, I suppose that's just as good.  I do wonder if this is really the right directory for these tests at all.  Its lit.local.cfg has a %clang_cc1 substitution that expands to a complaint that it shouldn't be used in Driver tests, so existing tests just use `%clang -cc1` instead.

Reply:
OK, I did that.  I used opt to discard the inline attribute that clang introduces.  If you know of a better way that doesn't perform inlining before the clang-linker-wrapper call, let me know.

Reply:
-disable-O0-optnone handles the optnone, don't think `noinline` affects that much.

Reply:
noinline prevents the inline pass in the test from running.

Reply:
I see, probably fine then.